src/compo/CMakeLists.txt

```diff
@@ -6,6 +6,7 @@ list(APPEND programs
   modis_aod2ioda.py
   aeronet_aod2ioda.py
   aeronet_aaod2ioda.py
+  VIIRS_BiasWriter.py
```

Comment:
Pedantic, but can you change this (and the filename) to be `viirs_biaswriter.py` so that it is consistent with the others?